src/writeData/components/ClientCodeQueryHelper.tsx

```diff
@@ -50,18 +50,19 @@ const ClientCodeQueryHelper: FC<Props> = ({clientQuery, contentID}) => {
       changeQuery(def.query)
       return
     }
-
-    const ast = parse(clientQuery)
-    const queryBucket = getBucketsFromAST(ast)[0]
-    if (queryBucket) {
-      changeBucket({name: queryBucket} as Bucket)
-    }
-    updateBucketInAST(ast, '<%= bucket %>')
-    let query = format_from_js_file(ast)
-    if (def.querySanitize) {
-      query = def.querySanitize(query)
-    }
-    changeQuery(query)
+    try {
```

Comment:
Not in the direct path of the errors seen in this ticket, but is also instances when lsp methods are not error handled.

There are remaining instances in notebook/flows code, which are not error handled. But since the error was not seen in notebooks (and there are several more instances there), I timeboxed to code outside of noteboooks/flows.